functests/10-test-template-validator-placement.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+SCRIPTPATH=$( dirname $(readlink -f $0) )
+source ${SCRIPTPATH}/testlib.sh
+
+RET=0
+TEST_NS="${KV_NAMESPACE}"
+
+oc create -n ${TEST_NS} -f "${SCRIPTPATH}/10-template-validator-unversioned-cr.yaml" || exit 2
+
+# Wait for the operator to create the deployment, we don't care if pods are actually ready, as
+# we only check for the pod scheduling fields
+sleep 100
```

Comment:
I think it may be better to call `oc get` in a loop with a timeout, instead of hard-coded `sleep`. But I don't know how fast the deployment is, or if it would save much time.

Reply:
I agree, will change taht

Reply:
@akrejcir done